app/views/announcement_mailer/notice.html.erb

```diff
@@ -2,23 +2,41 @@
   Hey<%= " all" if @emails.size > 1 %>,
 </p>
 
-<p>
-  We recently launched a new feature on HCB - organization announcements! This will help you send out announcements on HCB to people who follow your organization.
-  For more details, check out <%= link_to "our blog post", "https://blog.hcb.hackclub.com/posts/organization-announcements" %> and the new Announcements page on any of your organizations.
-</p>
+<% if @cancellation_email.present? %>
+  <p>
+    You recently received an email from HCB saying that your scheduled monthly announcement has been cancelled.
+    This is related to a new feature on HCB, organization announcements, which will help you send out announcements on HCB to people who follow your organization.
+  </p>
 
-<p>
-  Your organization also has monthly announcements enabled!
-  This is a feature that automatically generates an announcement each month to send out to your followers.
-</p>
+  <p>
+    Monthly announcements are an optional feature that automatically generates an announcement each month to send out to your followers.
+    By default, they will include a short message and public data about your organization from the past month such as donation and spending summaries, but you can edit the announcement to include whatever you'd like.
+  </p>
+
+  <p>
+    This feature was automatically enabled for <%= @event.name %>, which is why you were sent the email about your scheduled monthly announcement.
+    HCB will cancel sending monthly announcements if they are not edited or if there is no recent activity to include in the announcement.
+  </p>
+<% else %>
+  <p>
+    We recently launched a new feature on HCB - organization announcements! This will help you send out announcements on HCB to people who follow your organization.
```

Comment:
```suggestion
    We recently launched a new feature on HCB: organization announcements! This will help you send announcements to people who follow your organization on HCB.
```